iceoryx_examples/icedelivery/iox_publisher_with_options.cpp

```diff
@@ -29,6 +29,7 @@ static void sigHandler(int f_sig [[gnu::unused]])
 {
     // caught SIGINT or SIGTERM, now exit gracefully
     killswitch = true;
+    iox::runtime::PoshRuntime::getInstance().shutdown();
```

Comment:
@mossmaurice I think you have a PR where you moved this example somewhere else. After the merge this has to be adjusted accordingly.

Also, where should I document this?

Reply:
I guess we can merge #655 very soon. I think Git shoulnd't have issues detecting that this file was moved. I suppose there won't be any conflicts here.

Reply:
Do you know if there is a specific documentation for the blocking publisher option?